ci/pipelines/pr.yml

```diff
@@ -1058,8 +1058,20 @@ jobs:
             args:
               - -ec
               - |
```

Comment:
Not a blocker but you could potentially use `repository: alpine/git` instead of the maven image for this.

Reply:
ha yeah, I spotted that thinking yeah I'll change it why are we using maven.... I'll do it now!

Reply:
used the swiss army knife `concourse-runner`